playbooks/roles/mongo_3_0/tasks/main.yml

```diff
@@ -322,7 +322,6 @@
 - name: create mongodb users in a replica set
   mongodb_user:
     database: "{{ item.database }}"
-    login_database: 'admin'
```

Comment:
Does this break replica sets? Does removing this break single-instance deployments?